ClientGenerator/src/googleapis/codegen/languages/csharp/default/templates/_method.tmpl

```diff
@@ -134,11 +134,19 @@ public override string RestPath
 public Google.Apis.Download.IMediaDownloader MediaDownloader { get; private set; }
 
 /// <summary>Synchronously download the media into the given stream.</summary>
+[System.Obsolete("This method hides errors; use " + nameof(Download2) + "(), which returns the final download status.")]
```

Comment:
Not sure about making this obsolete. This would break a lot of code that's compiled with warnings-as-errors... I don't know whether its error-hiding issue is enough to warrant this. Hmm.